app/components/score_component.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class ScoreComponent < ViewComponent::Base
+  NOT_SCORED = -2
```

Comment:
Out of interest, why have you set NOT_SCORED = -2? Is this just to be low enough not to accidentally conflict with low scores? Or do the policy team populate the spreadsheet with -2 if a category hasn't been scored?